pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1629,6 +1629,26 @@ func validateDisks(field *k8sfield.Path, disks []v1.Disk) []metav1.StatusCause {
 			})
 		}
 
+		ioModeOK := false
+		switch disk.IO {
```

Comment:
Just suggestion, use alias type.

Reply:
Not sure what you mean here

Reply:
const Native DriverIO = "native".... and you can then use it to compare... (nit)

Reply:
@danielBelenky take a look at how we've handled cache modes validation, just a couple of lines above.
Let's make it similar. Please also define specific values that disk.IO may accept.